src/App.css

```diff
@@ -1,5 +1,39 @@
+body {
+  background-color: #E5E5E5;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  margin: 0;
+  min-height: 100%;
+}
+
 .App {
+  align-items: center;
+  display: flex;
   font-family: "Helvetica New", Helvetica, sans-serif;
+  height: 100vh;
+  justify-content: center;
+  text-align: center;
+}
+
+.Button {
+  background: #0577CA;
+  border-radius: 5px;
+  color: #FFFFFF;
+  font-size: 18px;
+  font-weight: Bold;
+  left: 0;
+  line-height: 49px;
+  margin-left: Auto;
+  margin-right: Auto;
+  position: Absolute;
```

Comment:
stylenit: please use lowercase for bold and absolute

Reply:
Woops, my bad.